package.json

```diff
@@ -115,7 +115,7 @@
     "clipboard": "^2.0.4",
     "cors-anywhere": "^0.4.1",
     "dynamic-pub-path-plugin": "0.0.4",
-    "gyronorm": "^2.0.6",
+    "gyronorm": "git+https://github.com/dorukeker/gyronorm.js.git",
```

Comment:
Is this necessary? The npm version seems to be the same as the GitHub one. And installing from GitHub via npm is known for creating unexpected obstacles.

Reply:
Didn't notice this change, will return it to 2.0.6 version

Reply:
Getting error, when using ^2.0.6.